region/lowernorfair/east.json

```diff
@@ -1508,45 +1691,96 @@
-                    "Charge",
-                    "Plasma",
-                    {"heatFrames": 600}
+                    {"enemyKill":{
+                      "enemies": [
+                          ["Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"], 
+                          ["Yellow Space Pirate (wall)"]
+                      ],
+                      "explicitWeapons": [ "Charge+Plasma" ]
+                    }},
+                    {"heatFrames": 450}
                   ]
                 },
                 {
                   "name": "Supers Pirates Kill",
                   "notable": false,
+                  "requires": [
+                    "canHeatRun",
+                    {"or": [
+                      {"and": [
+                        {"enemyKill":{
+                          "enemies": [
+                            ["Yellow Space Pirate (wall)"], 
+                            ["Yellow Space Pirate (wall)"]
+                          ],
+                          "explicitWeapons": [ "Super" ]
+                        }},
+                        {"heatFrames": 395}
+                      ]},
+                      {"and": [
+                        {"enemyKill":{
+                          "enemies": [
+                            ["Yellow Space Pirate (wall)"], 
+                            ["Yellow Space Pirate (wall)"], 
+                            ["Yellow Space Pirate (wall)"]
+                          ],
+                          "explicitWeapons": [ "Super" ]
+                        }},
+                        {"heatFrames": 330}
+                      ]}
+                    ]}
+                  ],
+                  "note": "The bottom wall pirates can be killed while stacked to save ammo."
+                },
+                {
+                  "name": "Missiles Pirates Kill",
+                  "notable": false,
                   "requires": [
                     "canHeatRun",
                     {"enemyKill":{
                       "enemies": [
-                        [
-                          "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"
-                        ]
+                        ["Yellow Space Pirate (wall)"], 
+                        ["Yellow Space Pirate (wall)"], 
+                        ["Yellow Space Pirate (wall)"]
                       ],
-                      "explicitWeapons": [ "Super" ]
+                      "explicitWeapons": [ "Missile" ]
                     }},
-                    {"heatFrames": 450}
-                  ]
+                    {"heatFrames": 600}
+                  ],
+                  "note": "The bottom wall pirates can be killed while stacked to save ammo, but it is not expected."
                 },
                 {
                   "name": "Full Spazer Pirates Kill",
                   "notable": false,
                   "requires": [
                     "canHeatRun",
```

Comment:
Just noticed, there are several `canHeatRun`. Should they be `h_canNavigateHeatRooms`?

Reply:
They don't need to be.  But it is safer if they were.
I think it better fits that they all be h_canNavigateHeatRooms.  heatProof doesn't give a mechanical advantage to performing a heat room strat, unlike with `canSuitlessMaridia`